policies/secure-development-lifecycle.md

```diff
@@ -0,0 +1,45 @@
+# Secure Development Lifecycle Policy
+
+## Purpose
+The secure development lifecycle describes the approach that is used to define, develop, deliver and maintain software. 
+
+## Scope
+This lifecycle is intended to be used by software teams including product managers, designers, developers and quality assurance to develop secure software products. This policy describes the lifecycle of feature development.
+
+## Policy
+This lifecycle integrates security into all aspects of application software development which is comprised of six phases. 
+
+1. Requirements Gathering and Planning:
+  - Identify the scope, goals, and necessary resources for development
+  - Define functional and security requirements for the feature 
+    - Considering core engineering principles (link)
+    - Security controls such as authentication, data encryption, and secure communication protocols
+
+2. Design:
+- Create a feature description that incorporates identified security controls and meets definition of ready (link)
+- Perform threat modeling to identify potential risks and design appropriate countermeasures
```

Comment:
Do we have a specific threat modelling framework in mind that we want to specify here? Or is it out of scope for this document?
https://www.hackerone.com/knowledge-center/threat-modeling-process-frameworks-and-tools

Reply:
We want to leave it open to the teams. For mobile, extension and core repos the plan is to add the 4 question framework into the templates. This will be the start.